catalog/policies/read-only.login.rego

```diff
@@ -0,0 +1,10 @@
+package spacelift
+
+username := input.session.login
+
+# case sensitive names of collaborators
+view_only_collaborators := { "nitrocode", "Nuru", "osterman", "aknysh" }
```

Comment:
I do t think we should publish this. It's not generic enough. If anything, it needs to call out it's an example or be parameterized

Reply:
we need to work with GitHub teams, not users (we can use some names for GitHub teams here as examples)